api/gateway_test.go

```diff
@@ -134,3 +142,4 @@ func TestBlockBootstrap(t *testing.T) {
 		time.Sleep(time.Millisecond)
 	}
 }
+*/
```

Comment:
hmm. I think I could actually uncomment it now, although it's really more of a consensus test than a gateway test.